pkg/cue/process/handle.go

```diff
@@ -139,6 +146,11 @@ func (ctx *templateContext) SetConfigs(configs []map[string]string) {
 	ctx.configs = configs
 }
 
+// SetParameters sets templateContext parameters
+func (ctx *templateContext) SetParameters(params map[string]interface{}) {
+	ctx.parameters = params
+}
```

Comment:
can you help add tests to cover these lines

Reply:
Ah, interesting! I would have to think of a way. I just wonder why they're not covered by the fact that SetParameters is invoked in [pkg/cue/process/handle_test.go](https://github.com/neinkeinkaffee/kubevela/blob/d4bbd139dfbedf25990536191673900c5c7698d0/pkg/cue/process/handle_test.go#L83)...?

Reply:
I can't see another test where the corresponding methods of the Context interface are covered, apart from handle_test.go. Could you suggest a good place where I might place an additional test for `SetParameters`?

Reply:
Oh, then it's ok, let me merge it